examples/jsm/nodes/core/Node.js

```diff
@@ -100,6 +100,28 @@ Node.prototype = {
 
 	},
 
+	getHash: function() {
```

Comment:
There are multiple formatting issues in this method.

Reply:
I did not think that formatting the JSON was necessary... you can post an example of the style?

Reply:
I can clean things up after merging. Don't be worry about this 👍 .